Bottom: 9d5e87ec3982847e89b8102eccac7589d928ca56
Top:    4d6be17bf8ca64d86251067d0cb6703460185ee2
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-01-11 14:37:50 +0000

Refresh of tmp-alterartions-to-build-all

---

diff --git a/w32/build-all.sh b/w32/build-all.sh
index 04aec82..e31e529 100755
--- a/w32/build-all.sh
+++ b/w32/build-all.sh
@@ -2,9 +2,6 @@ NEVERBUILD="
 "
 
 NOBUILD="
-"
-
-BUILD="
 jpeg
 zlib
 libpng
@@ -15,6 +12,10 @@ libiconv
 gettext
 glib
 atk
+"
+
+BUILD="
+freetype
 cairo
 pango
 gtk+
